package.json

```diff
@@ -46,6 +46,7 @@
     "react-transition-group": "2.5.3",
     "reactstrap": "6.5.0",
     "regenerator-runtime": "^0.13.7",
+    "sodium": "^3.0.2",
```

Comment:
How come sodium is needed in the dependencies now?

Reply:
I don't know why but it's needed when running the `scripts/package.sh` so I guess it's for the windows or linux builds. It was needed for the previous WRW version `3.1.1` but I had just installed it manually before running the build script. Including it in the dependencies saves doing that step.